apiary.apib

```diff
@@ -81,10 +81,38 @@ Any response that does not provide a ```200 OK``` response code should be treate
             X-Dude-Api-Server: Api Server ID
     + Attributes (Heartbeat)
 
-# Group Files
+# Group Import
 The assets endpoints allows for the access and management of asset import resources.
 
-## Asset Import Detail [/accounts/{accountId}/import/assets/import/{id}]
+## Asset Import Collection [/accounts/{accountId}/import{?page,pagesize,orderby,includeAudit}]
+This endpoint allows for the bulk retrieval of asset import jobs
+
++ Parameters
+    + accountId: 2 (number, required) - {INT} The account ID the resource belongs to.
+    + page: 1 (number, optional) - {INT} The page to request from the collection.
+    + pagesize: 1000 (number, optional) - {INT} The number of records to load in a page from the collection.
+    + orderby: AncestorIdPath asc (string, optional) - A string describing the ordering to perform on the collection.
+    + includeAudit: true (boolean, optional) - Flag to return audit information from the api
+
+### GET
++ Response 200 (application/json)
+    + Attributes (Base Collection Response)
+        + `data` (array[Asset Import], required)
+
+## Asset Import Create [/accounts/{accountId}/import/assets]
+This endpoint allows for an import to be created and scheduled
+
++ Parameters
+    + accountId: 2 (number, required) - {INT} The account ID the resource belongs to.
+    + file: data (file, required) - File to upload.
```

Comment:
Apparently, having an arbitrary type called "file" will trigger the correct type to generate